podium-gateway/src/main/resources/config/application.yml

```diff
@@ -136,6 +136,9 @@ podium:
         license-url:
     ribbon:
         display-on-active-profiles: dev
+    files:
+        max-size: 50
+        upload-dir: "/tmp/podium_data"
```

Comment:
A bit dangerous to use the `/tmp` directory for uploads, perhaps. Of course it is configurable, but at least for development I expect my uploaded files to be persistent. So, maybe make it `./upload` or `./podium_data` for the dev environment.
Are the quotes needed?